docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -776,8 +800,25 @@ The `addCustomMethods` function also adds the specified methods to the [element
 
 ```js
 const myTable = Selector('.my-table').addCustomMethods({
-    getCellText: (table, rowIndex, columnIndex) =>
-        table.rows[rowIndex].cells[columnIndex].innerText
+    getCellText: (table, rowIndex, columnIndex) => {
+        return table.rows[rowIndex].cells[columnIndex].innerText;
+    };
+});
+
+await t.expect(myTable.getCellText(1, 1)).contains('hey!');
+```
+
+If you use TypeScript, declare a Selector interface extension with your custom methods:
+
+```csharp
```

Comment:
There is a problem with jekyll and the `redcarpet` library that generates html-markup from '.md' files for our documentation site. It ignores `ts`or `typescript` for some reason (but it seems it should work) and generates the code as a plain text without code highlighting. I've spent some time in investigation but couldn't find the reason. 
`csharp` looks ok for typescript code highlighting, so I think we can use it for a while to don't waste time now.
(typescript is highlighted as `js` in a wrong way with `redcarpet` too because of the type casting syntax)